plugins/poll/assets/stylesheets/common/poll.scss

```diff
@@ -28,6 +28,57 @@ div.poll-outer {
       }
     }
 
+    .irv-poll-options {
+      display: flex;
+      flex-direction: column;
+      gap: 0.5em;
+      padding: 0.5em;
+
+      .irv-poll-option {
+        display: inline-flex;
+        details {
+          padding: 0;
+        }
+        summary {
+          min-width: 100px;
+        }
+        flex-wrap: wrap;
+        align-items: center;
+        gap: 0.5em;
+        padding: 0.5em;
+      }
+    }
+
+    .tabs {
+      display: none;
+    }
+
+    .discourse-poll-irv-results {
+      .tabs {
+        &.nav-items {
+          display: flex;
+          gap: 0.5em;
+          button {
+            background-color: var(--secondary);
+            border: none;
+            color: var(--primary-medium);
+          }
+          .active {
+            button {
+              color: var(--primary);
+            }
+          }
+        }
+      }
+      span.rank {
```

Comment:
do we need span here?

Reply:
no we don't: https://github.com/discourse/discourse/pull/27155/commits/98044fc11f9ea31269beb420591a22a53eab904f